conf/broker.conf

```diff
@@ -1149,6 +1149,10 @@ managedLedgerOffloadDeletionLagMs=14400000
 # (default is -1, which is disabled)
 managedLedgerOffloadAutoTriggerSizeThresholdBytes=-1
 
+# The number of seconds before triggering automatic offload to long term storage
+# (default is -1, which is disabled)
+managedLedgerOffloadThresholdInSeconds=-1
```

Comment:
`managedLedgerOffloadAutoTriggerThresholdInSeconds` to keep sync with `managedLedgerOffloadAutoTriggerSizeThresholdBytes`?

Reply:
Ok, make sense.